contracts/MemeFactory.sol

```diff
@@ -107,7 +107,7 @@ abstract contract MemeFactory {
     // Uniswap V3 fee tier of 1%
     uint24 public constant FEE_TIER = 10_000;
     /// The minimum tick that corresponds to a selected fee tier
-    int24 public constant MIN_TICK = -887200;
+    int24 public constant MIN_TICK = -887200; // TODO: double check
```

Comment:
@kupermind and @Sfgangloff  please double check that bound

Reply:
We will once again, but we already did - it does not work with the tick spacing otherwise.